phoenix-pherf/src/main/java/org/apache/phoenix/pherf/Pherf.java

```diff
@@ -76,6 +78,8 @@
         options.addOption("d", "debug", false, "Put tool in debug mode");
         options.addOption("stats", false,
                 "Update Phoenix Statistics after data is loaded with -l argument");
+		options.addOption("label", true, "Label a run. Result file name will be suffixed with specified label");
```

Comment:
Is this an override to then dynamically generated label? Also, it would be good if the label was applied as a column value in the result data. That way we can distinguish between runs on analysis.